app/helpers/feeds_helper.rb

```diff
@@ -0,0 +1,61 @@
+module FeedsHelper
+  def episode_offset_options
+    [
+      ["5 minutes", 300],
```

Comment:
I'd probably leave the values in here, and move the english strings to en.yml?  (Or i'm fine with tackling the i18n stuff in a separate ticket).